eZ/Publish/Core/REST/Server/Controller/Views.php

```diff
@@ -48,12 +48,20 @@ public function createView(Request $request)
             $method = 'findContent';
         }
 
+        $languageFilter = [
+            'languages' => null !== $viewInput->languageCode ? [$viewInput->languageCode] : Language::ALL,
+            'useAlwaysAvailable' => $viewInput->useAlwaysAvailable ?? true,
+        ];
+        if (!empty($viewInput->query->query->value)) {
```

Comment:
`->query->query->value` is not very descriptive. Expose first level as a local variable or make a private checker method (`isSomething`/`hasSomething`) for a whole `if` condition.